langchain/vectorstores/chroma.py

```diff
@@ -87,12 +89,22 @@ def __init__(
                     )
             self._client = chromadb.Client(self._client_settings)
 
-        self._embedding_function = embedding_function
+        if embeddings is None and "embedding_function" in kwargs:
+            # Use kwarg embedding_function and print a deprecation warning
+            assert isinstance(kwargs["embedding_function"], Embeddings), \
```

Comment:
to validate user input lets not use asserts but rather 
```
if not isinstance(kwargs["embedding_function"],...):
    raise ValueError
```

Reply:
@hwchase17 Done! 👍